articles/hardware/hs64/configuration.md

```diff
@@ -0,0 +1,19 @@
+---
+uid: hs64_configuration
+title: Headstage64 Configuration
+hardware: Headstage64
+configuration: true
+operator: ConfigureHeadstage64
+dataRate: 20.6
+timeUntilFullBuffer: 200 μs
+blockReadSize: 2048
```

Comment:
is this blockReadSize true?

Reply:
yes but I had to recalculate some other numbers there.